drivers/filesystems/fastfat_new/create.c

```diff
@@ -629,7 +629,7 @@ Return Value:
 
     ULONG MatchFlags = 0;
 
-    CCB LocalCcb;
+    CCB LocalCcb = {0};
```

Comment:
What problem does this fix? Is it fixing an uninitialized variable usage?

Reply:
Yes, uninitialized usage

Reply:
This may be worth going into a separate PR to be committed asap.

Reply:
Ok

Reply:
Yes please commit that line via a separate commit/PR!

Reply:
Is it still the case? Seems like nobody else spotted this, maybe that was MSVC RTC bug we have hacked around recently?

Reply:
Here my PR https://github.com/reactos/reactos/pull/3934

Reply:
That PR has been closed and superseded by a more correct approach by Timo. Can you remove that single-line change now here? Is the problem gone now that you intended to address with that line?

Reply:
PR https://github.com/reactos/reactos/pull/3909 fix my problem